templates/taxi/car_list.html

```diff
@@ -1,13 +1,24 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a href="{% url "taxi:car-create" %}">+</a></h1>
   {% if car_list %}
     <ul>
       {% for car in car_list %}
         <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
+          <tr>
+            <td>
+              <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
```

Comment:
There is an extra space in the URL tag: `{% url "taxi:car-detail" pk=car.id %} `. Remove the space after the URL tag to ensure proper URL generation.